src/components/Taxonomy/ActiveTopicConnection.tsx

```diff
@@ -7,15 +7,35 @@
  */
 
 import { useTranslation } from "react-i18next";
-import styled from "@emotion/styled";
-import { misc, spacing, fonts, colors } from "@ndla/core";
 import { DeleteBinLine } from "@ndla/icons/action";
-import { IconButton } from "@ndla/primitives";
+import { Button, IconButton, ListItemContent, ListItemHeading, ListItemRoot } from "@ndla/primitives";
+import { styled } from "@ndla/styled-system/jsx";
 import { Node } from "@ndla/types-taxonomy";
 import Breadcrumb from "./Breadcrumb";
 import RelevanceOptionSwitch from "./RelevanceOptionSwitch";
 import { MinimalNodeChild } from "../../containers/ArticlePage/LearningResourcePage/components/LearningResourceTaxonomy";
 
+const StyledWrapper = styled("div", {
+  base: {
+    display: "flex",
+    alignItems: "center",
+    gap: "3xsmall",
+  },
+});
+
+const StyledPrimaryConnectionButton = styled(Button, {
+  base: {
+    opacity: "0.3",
+    _hover: { opacity: "1" },
+    _focusVisible: { opacity: "1" },
+  },
+  variants: {
+    primary: {
+      true: { opacity: "1" },
+    },
+  },
+});
```

Comment:
Beholdt denne stylingen for å holde det så likt som mulig, alternativt kan vi sette en annen variant!